### PR TITLE
V1.1 public review doc

### PR DESCRIPTION
**Pull request by woodbe**
The overview document for the public review 1 period.

### CODE REVIEW

File: v1.1 Review/Public_review_1.1.adoc [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+A concern has been raised that mobile devices (the first target for the BIOPPM) are not normally tested/reviewed against the public quality measures and usually focus on a balance that meets the FAR/FRR needs of the device, and that the public measures are not designed for mobile devices in general, but other systems that may share biometric data.
+
+The BIOSD has a section with a proposed outline of what needs to be provided to the evaluator for review, and is looking for comments on this section to better target it to both what a validator would expect to see as complete and also what a vendor would consider appropriate given the nature of the quality measures.
+
+It is important to note that this information is needed for testing that the samples provided to the biometric system have the information needed to build a template and then to verify a new sample to the template. So the focus is on how to create poor samples and how to ensure they are discarded from use.
```
Comment (gregott) [inline v1.1 Review/Public_review_1.1.adoc]: I am not sure what is meant by the phrase "and then to verify a new sample to the template."  Are you saying that templates built from new biometric samples have enough information to build a template that can then be used for a match determination to an enrolled template?

Comment (woodbe) [reply v1.1 Review/Public_review_1.1.adoc]: Something along those lines, like what Greg F was pointing out that the focus here is on how to determine quality to be able to create low quality samples. I'm happy to change the wording here if you have a suggestion (I was a little at a loss in better phrasing).

Comment (gregott) [reply v1.1 Review/Public_review_1.1.adoc]: How about something along these lines:  It is important to note that this information is needed for testing that the samples provided to the biometric system have sufficient information needed to build templates which will result in the desired performance, i.e. acceptable error rates.  So the focus is on how to create poor samples and how to ensure they are discarded from use.

**Comment by woodbe** [comment]
@gregott I just posted that change. Thank you for the comment.

**Review by n-kai — CHANGES_REQUESTED** [review]
Could you correct the following errors?

- "Link" in Table 1. Review Documents are not correct
- Delete "cYEL" in the 2nd box of first row in the figure under Document Release Process

**Comment by woodbe** [comment]
@n-kai thank you for the review. The links will work once I publish everything, those are where I will put the docs (but I hadn't generated anything or uploaded them yet).

As for the cYEL, that is a code in the ditaa that will make that box yellow once I product the final output. When we do the next review period (so the next version of this document for the next review period), I will move that cYEL into the next review period box so that one will be yellow, to show which step we are on.

**Pull request merged**